src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -315,7 +315,7 @@ JAVADOC -> <audio><source src="horse.ogg" type="audio/ogg"/></audio><EOF> [0:0]
 |       |   `--CLOSE -> > [0:6]
 |       |--HTML_ELEMENT -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
 |       |   `--SINGLETON_ELEMENT -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
-|       |       `--SINGLETON_TAG -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
+|       |       `--EMPTY_TAG -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
```

Comment:
I created separate issue on this - #4448 .
Should be done when we come to checkstyle8 version